sale_product_set/models/product_set.py

```diff
@@ -1,11 +1,20 @@
 # -*- encoding: utf-8 -*-
-from openerp import fields, models, _
+from openerp import api, fields, models, _
 
 
 class ProductSet(models.Model):
     _name = 'product.set'
     _description = 'Product set'
 
-    name = fields.Char(_('Name'), help=_('Product set name'), required=True)
+    name = fields.Char(u"Name", help=u"Product set name", required=True)
```

Comment:
Not absolutely needed on this particular example, indeed, but I believe it's a safe practice to always put unicode objects on things that are targeted to be read by humans, and simple strings for things that are read by the machines.